test/web_console/middleware_test.rb

```diff
@@ -20,7 +20,7 @@ def call(env)
     setup do
       Request.stubs(:whitelisted_ips).returns(IPAddr.new('0.0.0.0/0'))
 
-      @app = Middleware.new(Application.new)
+      @app = Middleware.new(Application.new, { mount_point: '/' })
```

Comment:
Actually, if you use a different `mount_point`, without changing the current tests, will they pass?